### PR TITLE
[WIP] Country page

### PR DESCRIPTION
**Pull request by danielfdsilva**

### CODE REVIEW

File: app/assets/scripts/views/admin-area.js [hunk 1]
```diff
@@ -491,7 +491,7 @@ class AdminArea extends React.Component {
 
   render () {
     return (
-      <App className='page--region'>
+      <App className='page--country'>
```
Comment (danielfdsilva) [inline app/assets/scripts/views/admin-area.js]: @ascalamogna This component is used both for the country and the region. Could we use `page--admin-area` as a class so it's clearer and works for both?

Comment (ascalamogna) [reply app/assets/scripts/views/admin-area.js]: @danielfdsilva there's specific styles needed for the country page, I modified the layout a bit so it needs a separate class.

**Comment by danielfdsilva** [comment]
Ok. I'll make the class dynamic then, otherwise the region page has the
same class

**Comment by danielfdsilva** [comment]
@ascalamogna fixed. Please test and merge at will. :)

**Pull request merged**